components/domains/registerV2.tsx

```diff
@@ -213,14 +234,28 @@ const RegisterV2: FunctionComponent<RegisterV2Props> = ({ domain }) => {
     hasMainDomain,
     address,
     sponsor,
+    metaHash,
   ]);
 
   useEffect(() => {
-    if (!registerData?.transaction_hash) return;
+    if (!registerData?.transaction_hash || !salt) return;
     posthog?.capture("register");
-    addTransaction({ hash: registerData?.transaction_hash ?? "" });
+    // register the metadata to the sales manager db
+    const requestOptions = {
+      method: "POST",
+      body: JSON.stringify({
+        meta_hash: computeMetadataHash(email, usState, salt),
```

Comment:
You should use the state that you setted

Reply:
Correct